js/foundation.slider.js

```diff
@@ -1,7 +1,14 @@
 'use strict';
 
-!function($) {
+import $ from 'jquery';
+import { Keyboard } from './foundation.util.keyboard';
+import { Move } from './foundation.util.motion';
+import { GetYoDigits, rtl } from './foundation.util.core';
+let Rtl = rtl;
```

Comment:
Not sure if this works with ES2015 and Webpack to import with `import { GetYoDigits, rtl as Rtl } from './foundation.util.core';` or if only TypeScript supports this